data/pins_sonata.xdc

```diff
@@ -0,0 +1,85 @@
+## Copyright lowRISC contributors.
+## Licensed under the Apache License, Version 2.0, see LICENSE for details.
+## SPDX-License-Identifier: Apache-2.0
+
+## Clocks
+create_clock -period 40.000 -name mainclk -waveform {0.000 20.000} [get_ports main_clk]
+create_clock -period 100.000 -name tck -waveform {0.000 50.000} [get_ports tck_i]
+
+## Reset
+set_property PACKAGE_PIN R11 [get_ports {nrst_btn}]
+set_property IOSTANDARD LVCMOS33 [get_ports {nrst_btn}]
+
+## General purpose LEDs
+set_property -dict { PACKAGE_PIN B13 IOSTANDARD LVCMOS33 } [get_ports {led_user[0]}];
+set_property -dict { PACKAGE_PIN B14 IOSTANDARD LVCMOS33 } [get_ports {led_user[1]}];
+set_property -dict { PACKAGE_PIN C12 IOSTANDARD LVCMOS33 } [get_ports {led_user[2]}];
+set_property -dict { PACKAGE_PIN B12 IOSTANDARD LVCMOS33 } [get_ports {led_user[3]}];
+set_property -dict { PACKAGE_PIN B11 IOSTANDARD LVCMOS33 } [get_ports {led_user[4]}];
+set_property -dict { PACKAGE_PIN A11 IOSTANDARD LVCMOS33 } [get_ports {led_user[5]}];
+set_property -dict { PACKAGE_PIN F13 IOSTANDARD LVCMOS33 } [get_ports {led_user[6]}];
+set_property -dict { PACKAGE_PIN F14 IOSTANDARD LVCMOS33 } [get_ports {led_user[7]}];
```

Comment:
Could I ask that port names at the FPGA-specific top level be consistent with the signal names on the schematics, please? I recognise that the use of capitals may be aesthetically offensive to some, and I'd say that preserving case doesn't matter since most search functions are case-insensitive.

When someone is new to a project, the extra couple of levels of indirection and renaming can make it hard for them to pair up the RTL with the circuit schematics; they need to know that the '.xdc' file exists, and where it lives, and then to map RTL port name to FPGA pin, and FPGA pin on the schematic to signal name/device on the schematic by searching for the pin name.

Right now with simple I/O it is less of an issue. As the design grows, the more complex devices are added, and the connectivity is increased, I think it's helpful for everything to be consistent.

This presently concerns the 'usr_jtag_...' signals, 'usrled..' signals, 'usrsw..' signals and 'navsw_..' signals (as named on the schematics).